app/helpers/embed_player_helper.rb

```diff
@@ -3,6 +3,7 @@ module EmbedPlayerHelper
 
   EMBED_PLAYER_LANDING_PATH = "/listen"
   EMBED_PLAYER_PATH = "/e"
+  EMBED_PLAYER_PREVIEW_PATH = "/preview"
```

Comment:
Not exactly sure where this is happening - but this doesn't appear to fix the "user copies a url with their JWT in it" bug.  In fact, even the not-in-iframe fields at the bottom of the page have a JWT in them.

We want to prevent user's from every copying/using a url that looks different from after the episode publishes.

If I go to an unpublished episode, i see it loading `/preview` as expected.  But:

- In the iframe, "Download Audio" has a JWT.  This isn't great - I'd rather hide or give them a url that 404s with dovetail until the episode publishes.
- In the iframe "Copy Embed Code" has a JWT.  This is a problem.
- Outside the iframe, the "Embeddable Player Link" field has a JWT.  It should not, and neither should the copy link. And we should hide the "open in new tab" link until the episode has published.
- Outside the iframe, the "Embeddable Player Iframe" field has the JWT (same problems as above)
- Outside the iframe, the "Media Link" seems correct (you get a 404 from DT until it publishes).

Reply:
Since Play is now sending back the embed URL and iframe markup, we'll have to add some logic to remove the JWT from the audio URL when generating those, and only use it for audio playback. Unpublished audio shouldn't be getting the share buttons to copy ULR or embed code in the iframe. I'll check to see where the disconnect is that is still showing those buttons.

Reply:
Kind of wish the Play PR hadn't been merged till this PR was reviewed and approved. The two PR's were tightly coupled. :(